examples/deploy_k8s_beta/workspace.yaml

```diff
@@ -0,0 +1,4 @@
+load_from:
+  - python_file:
+      relative_path: iris_analysis/definitions.py
+      location_name: iris_analysis
```

Comment:
only needed for local dev (we could use a pyproject.toml too)

Reply:
wil remove